runtime/commands/default.js

```diff
@@ -11,9 +11,9 @@ Commands.ping = {
   timeout: 10,
   level: 0,
   fn: function (msg) {
-    var initTime = Date.now()
+    var initTime = new Date(msg.timestamp)
```

Comment:
Why parse them to a date object?
The timestamp is in milliseconds anyway so why not subtract them from eachother?